### PR TITLE
Add gap between nav entry

### PR DESCRIPTION
**Pull request by CarlSchwan**
Before:

image

After:

image

### CODE REVIEW

File: src/components/NcAppNavigation/NcAppNavigation.vue [hunk 1]
```diff
@@ -177,6 +177,7 @@ export default {
 		box-sizing: border-box;
 		display: flex;
 		flex-direction: column;
+		gap: 4px;
```
Comment (jancborchardt) [inline src/components/NcAppNavigation/NcAppNavigation.vue:180]: The variable `--default-grid-baseline` was added by @juliushaertl, it can be used here. :)

**Review by jancborchardt — CHANGES_REQUESTED** [review]

File: src/components/NcAppNavigation/NcAppNavigation.vue (hunk 1 above)
Comment (CarlSchwan) [reply src/components/NcAppNavigation/NcAppNavigation.vue]: Done

**Review by marcoambrosini — APPROVED** [review]
These things make me very happy ❤️

**Review by jancborchardt — APPROVED** [review]
Great eye for design and details in the mockup @CarlSchwan :)

**Pull request merged**